src/ch05/15.py

```diff
@@ -0,0 +1,36 @@
+import pylab
+import numpy as np
+from random import *
+pi=np.pi
+n=1000
+ts=np.linspace(0,12*pi,n)
+xs=np.sin(ts)*(np.exp(np.cos(ts))-2*np.cos(4*ts)-np.sin(ts/12)**5)
+ys=np.cos(ts)*(np.exp(np.cos(ts))-2*np.cos(4*ts)-np.sin(ts/12)**5)
+
+
+map=[[0x8,0x8,0x7f,0x49,0x7f,0x8,0x8],[0x8,0x8,0x3e,0x2a,0x7f,0x14,0x63]]
+R,C=7,7
+ds=5
+#m=int(input("> "))
+m = 1
+ds=20
+pylab.axis("off")
+for r in range(R):
+    for L in range(len(map)):
+        for c in range(C-1,-1,-1):
+            if (map[L][r]>>c)%2:
+                a=2*pi*random()
+                sina=np.sin(a)
+                cosa=np.cos(a)
+                ra=(1/m)*(0.8+0.2*random())

```

Comment:
```suggestion
                ra=(1/m)*(2.0+0.2*random())
```

The radius $0.8 + X \sim \mathcal{U}_{[0, 0.2)}$ seems to be too small 🤔:

![image](https://user-images.githubusercontent.com/20769958/212463546-a4c848eb-5871-4776-8e26-51c7852ea593.png)

I think the radius should be enlarged. Like $2.0 + X \sim \mathcal{U}_{[0, 0.2)}$:

![image](https://user-images.githubusercontent.com/20769958/212463595-2bfbedfe-d527-4f91-b14d-a5096181c2ee.png)